yellowbrick/features/pca.py

```diff
@@ -118,7 +119,7 @@ def __init__(self,
             [('scale', StandardScaler(with_std=self.scale)),
              ('pca', PCA(self.proj_dim, random_state=random_state))]
         )
-
+        self.alpha=alpha
```

Comment:
This is exactly right.  Additionally, We should include alpha parameter in the docstring above

Reply:
Sure!